nnvm/tests/python/frontend/coreml/model_zoo/__init__.py

```diff
@@ -1,4 +1,4 @@
-import urllib
+from six.moves import urllib
```

Comment:
make sure six in installed as dependency in CI

Reply:
added. thanks for reminding.